### ISSUE TITLE
Generate the debug/temporary (Android) certificate once

### ISSUE THREAD

**Issue by Jerakin** [defold/editor2-issues#2532]
Currently when bundling (Android) without a certificate Defold adds a temporary certificate. It could be helpful if that certificate is continuously reused for the project, a user could then reinstall an app without needing to uninstall it first.

**Comment by erikangelin-king** [comment]
Workaround: generate and provide your own certificate

**Comment by Jerakin** [comment]
Yeah and I assume people who have worked with it before does it (and would continue to even if the feature is implemented) as it is super easy. More of a "beginner user friendly" thing. :)
